packages/beacon-node/src/chain/bls/multithread/jobItem.ts

```diff
@@ -70,7 +70,7 @@ export function jobItemWorkReq(job: JobQueueItem, metrics: Metrics | null): BlsW
       // and not a problem in the near future
```

Comment:
stale comment
deployed to `feat2` to gather info

Reply:
Removed